src/components/Blogcard.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+import React from 'react';
+import Box from './Box';
+import Text from './Text';
+
+const Blogcard = ({ imageUrl, url, title }) => {
+	return (
+		<Box mx="mxxxl" target="_blank" cursor="pointer" as="a" href={url}>
+			<Box
+				width={{ mobS: '34.3rem', tabS: '27.6rem', deskM: '38rem' }}
+				height={{ mobS: '20rem', tabS: '16.2rem', deskM: '22rem' }}
+				position="relative"
+				mb="mxl"
+				mt={{ mobS: 'wxs', tabL: 'mxxxl', deskM: 'ws' }}
```

Comment:
in principle ...we should put marginTop outside the component when we are using it...so remove it here and we will use this when needed

Reply:
yeah but in mobile state we need margin for every image